tests/functional/pages/surveys/census/communal/establishment-type.page.js

```diff
@@ -33,6 +33,11 @@ class EstablishmentTypePage extends MultipleChoiceWithOtherPage {
     return this
   }
 
+  setEstablishmentTypeAnswerOtherOtherText(value) {
```

Comment:
I guess merely adding a suffix probably isn't the best option then? Surely Other and OtherText would be a more sensible choice?